plugins/out_opensearch/opensearch.c

```diff
@@ -473,13 +520,13 @@ static int opensearch_format(struct flb_config *config,
                 }
             }
         }
-        else if (ctx->current_time_index == FLB_TRUE) {
+		else if (ctx->current_time_index == FLB_TRUE) {
```

Comment:
Did you use tabs instead of white spaces for indentations?
If so, we recommend to use white spaces for that purpose.

Reply:
Yeah I was using tabs. I'll fix this.